CHANGELOG.md

```diff
@@ -1,96 +1,111 @@
 ## [23.17.1](https://github.com/jest-community/eslint-plugin-jest/compare/v23.17.0...v23.17.1) (2020-06-23)
```

Comment:
This file is automatically generated, so should not be formatted with prettier as it'll get overridden the next time a release happens :)

Reply:
This was edited by husky commit tasks when I merged the latest changes with master 😓 
Removed them with https://github.com/jest-community/eslint-plugin-jest/pull/553/commits/ab2c365896fe7e61daa8dbd2fd4ccdae29ac5046